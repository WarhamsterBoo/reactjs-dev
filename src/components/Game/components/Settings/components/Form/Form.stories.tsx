import React from "react";
import { Form } from "./Form";

export default { title: "Form component" };

export const FormComponent: React.FC<{}> = () => {
  return <Form />;
};
