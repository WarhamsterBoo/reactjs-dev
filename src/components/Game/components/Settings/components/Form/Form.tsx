import React from "react";

export const Form: React.FC<{}> = () => {
  return <div>I'm form</div>;
};
