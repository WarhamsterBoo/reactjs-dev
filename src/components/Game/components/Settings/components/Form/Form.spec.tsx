import React from "react";
import renderer from "react-test-renderer";
import { Form } from "./Form";

describe("Form", () => {
  it("should render", () => {
    const sut = <Form />;

    expect(renderer.create(sut).toJSON()).toMatchSnapshot();
  });
});
