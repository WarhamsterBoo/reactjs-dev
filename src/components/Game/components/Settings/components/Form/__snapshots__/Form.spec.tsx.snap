// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render 1`] = `
<div>
  I'm form
</div>
`;
